### PR TITLE
15 workshop notifications countdown

### PR DESCRIPTION
**Pull request by rigoucr**
To test this functionality:
- Revert all features `drush fra -y`
- Create user with authenticated user role.
- Create user with drupal_administrator role.
- Create a Workshop with availability of 1 person.
- When the user registration is not complete try to apply to workshop.
- When the user registration is complete try apply to workshop.
  thanks.

### CODE REVIEW

File: sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.features.inc [hunk 1]
```diff
@@ -61,6 +68,94 @@ function dccr_ftr_workshop_flag_default_flags() {
...
+    "arguments" : null,
+    "message_text" : {
+      "en" : [
+        {
+          "value" : "The workshop [message:field-workshop-node:title] is full",
+          "format" : "plain_text",
+          "safe_value" : "\\u003Cp\\u003EThe workshop [message:field-workshop-node:title] is full\\u003C\\/p\\u003E\\n"
+        },
+        {
+          "value" : "The workshop [message:field-workshop-node:title] is full.\\n\\nThanks.",
```
Comment (kporras07) [inline sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.features.inc]: It looks bad that the email subject be exactly the same than the email body. 
Could you add something more useful information to the email body?

File: sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.features.inc [hunk 2]
```diff
@@ -61,6 +68,94 @@ function dccr_ftr_workshop_flag_default_flags() {
...
+        }
+      ],
+      "es" : [
+        {
+          "value" : "Usted se ha registrado en el taller de [message:field-workshop-node:title]\\n",
+          "format" : "plain_text",
+          "safe_value" : "\\u003Cp\\u003EUsted se ha registrado en el taller de [message:field-workshop-node:title]\\u003C\\/p\\u003E\\n"
+        },
+        {
+          "value" : "Querido [current-user:name]\\nUsted se ha registrado correctamente en el taller  [message:field-workshop-node:title] , sera el [message:field-workshop-node:field-workshop-date:value:custom:d\\/m\\/Y - g:i a]\\nGracias.\\n",
```
Comment (kporras07) [inline sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.features.inc]: I think "Estimado" is a better word than "Querido"

Comment (kporras07) [inline sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.features.inc]: You are registered in the workshop [message:field-workshop-node:title]; it's going to be on [message:field-workshop-node:field-workshop-date:value:custom:m\/d\/Y - g:i a]\r\n\r\nThanks.",

Comment (kporras07) [inline sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.features.inc]: "Estimado [current-user:name]\nUsted se ha registrado correctamente en el taller [message:field-workshop-node:title]; el mismo será el día [message:field-workshop-node:field-workshop-date:value:custom:d\/m\/Y - g:i a]\nGracias.\n",

**Comment by rigoucr** [comment]
thanks for the suggestions :D

**Comment by rigoucr** [comment]
Why these permissions for the role drupal administrator?
- Flush caches.
- Administer entity translation.
- Administer views.

**Pull request merged**